samples/original-manual-adjusted.rb

```diff
@@ -30,7 +30,7 @@ def dewikify_hi(str, terms, intro = false)
       str = str
         .gsub(CODE_RE) { |x| code << x; "CODE#[#{code.length-1}]" }
         .gsub(/#{Regexp.quote(terms)}/i, '@\0@')
-        .gsub(/CODE#\[(\d+)\]/) { code[$1.to_i] }
+        .gsub(/CODE#\[(\d+)\]/) { code[Regexp.last_match(1).to_i] }
```

Comment:
Yeah, I don't see a tidy way to keep the `case` with that... but I do have a solution. We don't even include `original-manual.rb` or `original-manual-adjusted.rb` in the working samples list, and we've extract the "real" manual over to its own gem in https://github.com/shoes/shoes-manual. How about we just get rid of these two files entirely and adjust the other instances to use `match`?